### DISCUSSION TITLE
1.3 Terminal Velocity Equation

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#23] [Q&A] [answered]
I'm confused on where this equation came from after solving for terminal velocity.

![image](https://user-images.githubusercontent.com/106542680/171423643-888ade49-9928-412c-8796-8e113c0a36fc.png)

**Comment by ecieszynski** [comment, marked as answer]
I didn't see that this question was already asked so disregard

**Reply by cooperrc** [reply]
> I didn't see that this question was already asked so disregard

Yes, closing the loop: you can find this discussion at #20
